Python/samples/README.md

```diff
@@ -66,12 +66,21 @@ Recognizers.recognize_temperature(user_input, culture),
 # DateTime recognizer - This function will find any Date even if its write in colloquial language
 # E.g "I'll go back 8pm today" will return "2017-10-04 20:00:00"
 Recognizers.recognize_datetime(user_input, culture)
-````
+```
 
-All these models accept `user_input: str` as a string and returns a **List** of [ModelResult](../libraries/recognizers-text/recognizers_text/model.py#L10-L16):
+All these models accept `user_input: str`, `culture: str` and returns a **List** of [`ModelResult`](../libraries/recognizers-text/recognizers_text/model.py#L11-L48) object:

```

Comment:
Shouldn't this say that culture is optional and if not provided English is used as default?

Reply:
Currently, somehow [the `culture` argument is not "optional"](https://github.com/microsoft/Recognizers-Text/blob/master/Python/libraries/recognizers-number/recognizers_number/number/number_recognizer.py#L149) (which I expected it is).
I think we should note that this is not optional in the `README.md` currently and give it default culture in the future. (maybe in another pull request?)

Reply:
#2410 issue